packages/explat-client-react-helpers/README.md

```diff
@@ -1,7 +1,65 @@
-**NOT READY FOR USE**
-
 # ExPlat Client: React Helpers
 
-This is the React Interface for the standalone client for Automattic's ExPlat.
+This is the React Interface for the standalone client for Automattic's ExPlat. To be used in conjunction with `@automattic/explat-client`, see also the README over there.
+
+## API: `<Experiment>`
+
+### Type signature
+
+`Experiment: (name: string, defaultExperience, treatmentExperience, loadingExperience) => JSX.Element `
+
+### Usage
+
+```
+<Experiment
+	name="experiment_name"
+	defaultExperience={ <DefaultComponent /> }
+	treatmentExperience={ <TreatmentComponent /> }
+	loadingExperience={ <LoadingComponent /> }
+/>;
+```
+
+- The simplest and safest way to experiement, but not useful everywhere.
+- For those not using typescript, make sure you provide all the Experience props.
+
+## API: `useExperiment('experiment_name')`
+
+### Type signature
+
+`useExperiment: (experimentName: string) => [boolean, ExperimentAssignment | null]`
+
+### Usage
+
+```
+const [ isLoadingExperimentAssignment, experimentAssignment ] = useExperiment( 'experiment_name' );
+
+if ( isLoadingExperimentAssignment ) {
+	// Show loading experience
+} else if ( experimentAssignment.variationName ) {
```

Comment:
Would it be better to show `=== 'treatment'`?

Reply:
Hmm, on reflection maybe